worlds/dlcquest/test/checks/world_checks.py

```diff
@@ -39,4 +39,4 @@ def assert_can_win(tester: DLCQuestTestBase, multiworld: MultiWorld):
 
 def assert_same_number_items_locations(tester: DLCQuestTestBase, multiworld: MultiWorld):
     non_event_locations = [location for location in multiworld.get_locations() if not location.advancement]
-    tester.assertEqual(len(multiworld.itempool), len(non_event_locations))
```

Comment:
The IDE generally recommends keeping the empty line at the end of the file

Reply:
This is adding a blank line, which is recommended

Reply:
Oh, is it? Maybe I misunderstood the git diff, it looked to me like the change was removing the line. Carry on then.